README.md

```diff
@@ -29,7 +29,7 @@ $ kubectl get service deis-workflow  # note the IP address for later
 Then open an SSH session to a Kubernetes minion node, install the `deis` client, and register:
 
 ```console
-$ curl -sSL http://deis.io/deis-cli/install.sh | sh
+$ curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | sh
```

Comment:
Let's change the pipe to `bash` so it'll work on Linux. I'd link to the issue on deis/workflow but I'm on mobile :)